lib/LMSManagers/LMSMessageManager.php

```diff
@@ -51,7 +51,7 @@ public function GetMessages($customerid, $limit = null)
                         $file_manager = new LMSFileManager($this->db, $this->auth, $this->cache, $this->syslog);
                     }
                     $file_containers = $file_manager->GetFileContainers('messageid', $message['id']);
-                    $message['files'] = $file_containers[0]['files'];
+                    $message['files'] = $file_containers['messageid'][0]['files'];
```

Comment:
Tutaj ustawianie pod indeksem takim jak typ zasobu nie jest potrzebne - to można zrobić na zewnątrz tylko przy netdevinfo.